server/lib/nl/config_builder/timeline.py

```diff
@@ -147,9 +156,13 @@ def single_place_multiple_var_timeline_block(column,
 
   # Line chart for the stat var
   tile = Tile(type=Tile.TileType.LINE, title=title, stat_var_key=[])
-  date_string = get_date_string(date)
+  if date_range:
+    start_date, end_date = get_date_range(date_range)
```

Comment:
this block seems to be repeated a couple of times. would a helper function be warranted?

Reply:
done

Reply:
Thanks!